torcharrow/test/transformation/test_text_ops.py

```diff
@@ -114,6 +116,19 @@ def setUpClass(cls):
                 ]
             ),
         )
+
+        cls.base_df_add_token = ta.dataframe(
+            {
+                "text": [["Hello", "world"], ["How", "are", "you!"]],
+                "indices": [[1, 2], [3, 4, 5]],
+            },
+            dtype=dt.Struct(
+                fields=[
+                    dt.Field("text", dt.List(dt.string)),
+                    dt.Field("indices", dt.List(dt.int64)),
```

Comment:
@bearzx  is right. `df["indices"]` is int64 but `add_tokens` expects `int32`, which cause type mismatch

cc @Nayef211

Reply:
This did the trick thanks @bearzx and @wenleix!